### PR TITLE
add <br> to string concat

### LINKED ISSUE #1974: Concatenate values with newlines rather than spaces in Lab Observations
## Action Requested
In Lab ObservationsWe want values (that had to be concatenated) to be concatenated using newlines instead of spaces. The current HTML view of eCRs separates the texts with newlines, but we've currently been doing spaces.
- **Example** for seed data `61f13d32-b25c-4fb6-998b-73de2dbf87f2`: 
- **Relevant code**: The `ConcatStrings` custom filter (which joins strings via `" "` instead of newlines) is applied to the returned `valueStringObject` in `Resource/_Observation.liquid`

## Acceptance Criteria

- [ ] For a given lab observation, value strings that had to be concatenated are joined using newlines instead of spaces.

## Additional Information
This ticket came out of PR #1921 (scope was getting too big so we moved addressing these issues into a new ticket)

### PR DESCRIPTION
**Pull request by nickbristow**
# PULL REQUEST

## Summary
Instead of concating strings with `" "` use `"<br/>"` instead

## Related Issue
Fixes #1974

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Review by BobanL — CHANGES_REQUESTED** [review]
Hide comment is showing up in every lab result, but everything else looks ok!

File: containers/orchestration/docker-compose.yml [hunk 1]
```diff
@@ -57,13 +57,15 @@ services:
     logging:
       driver: "json-file"
   trigger-code-reference-service:
+    platform: linux/amd64
```
[the remark this answers is not in the document]
Comment (nickbristow) [reply containers/orchestration/docker-compose.yml:60]: i had trouble loading it when i didn't have this set

File: containers/ecr-viewer/src/app/view-data/components/EvaluateTable.tsx [hunk 2]
```diff
@@ -98,13 +98,21 @@ const BuildRow: React.FC<BuildRowProps> = ({
...
       rowCellData = column.applyToValue(rowCellData);
-    } else if (!rowCellData) {
+    }
+
+    if (!rowCellData) {
       rowCellData = <span className={"text-italic text-base"}>No data</span>;
-    } else if (column.hiddenBaseText) {
+    }
+
+    if (rowCellData && column.hiddenBaseText) {
```
[the remark this answers is not in the document]
Comment (nickbristow) [reply containers/ecr-viewer/src/app/view-data/components/EvaluateTable.tsx]: i think i get what your saying? hows this

**Review by BobanL — APPROVED** [review]
👏

**Pull request merged**